Source/Core/VideoCommon/HiresTextures.cpp

```diff
@@ -91,32 +101,75 @@ void HiresTexture::Update()
 
   const std::string& game_id = SConfig::GetInstance().GetGameID();
   const std::string texture_directory = GetTextureDirectory(game_id);
-  std::vector<std::string> extensions{
-      ".png", ".bmp", ".tga", ".dds",
-      ".jpg"  // Why not? Could be useful for large photo-like textures
-  };
+  std::vector<std::string> extensions{".dds", ".png", ".bmp",  ".tga",  ".tgb",
+                                      ".jpg", ".jpe", ".jpeg", ".jfif", ".hdr"};
 
-  std::vector<std::string> filenames =
+  std::vector<std::string> filepaths_found =
```

Comment:
Do you meant file_paths then - I'm understanding this as if you want to completely get rid of filepath

I never saw on any PC any text separating filepath or filename in 2 words, so this is some special thing in dolphin then.